data/interfaces/default/graphs.html

```diff
@@ -327,7 +331,8 @@ <h4><i class="fa fa-calendar"></i> Total <span class="yaxis-text" style="text-tr
             'Direct Play': '#E5A00D',
             'Direct Stream': '#FFFFFF',
             'Transcode': '#F06464',
-            'Max. Concurrent Streams': '#96C83C'
+            'Max. Concurrent Streams': '#96C83C',
+            'Total': '#F06464'
```

Comment:
Red is already used by Music.
```suggestion
            'Total': '#96C83C'
```